### ISSUE TITLE
Baby MemNN Example: Poor Accuracy

### ISSUE THREAD

**Issue by anthonyhsyu** [keras-team/keras#13357]
**System information**  
- Have I written custom code (as opposed to using example directory): no
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Linux Ubuntu 16.04.5
- TensorFlow backend (yes / no): yes
- TensorFlow version: 1.14
- Keras version: 2.2.5
- Python version: 3.7.4
- CUDA/cuDNN version: (running on CPU)
- GPU model and memory: (running on CPU)

**Describe the current behavior**  
The example was found at [https://keras.io/examples/babi_memnn/](https://keras.io/examples/babi_memnn/). The example runs, and the training accuracy is close to the advertised 98.6%, but the validation accuracy remains between 0.5 and 0.6 throughout 120 epochs. 

**Describe the expected behavior**  
For an example, we'd expect a higher validation accuracy, as all this is showing is overfitting.

**Code to reproduce the issue**  
Copy the code from the example above and save it as a Python file. On command line, `cd` to the Python file, then run `python <filename.py>`.

**Other info / logs**  
This was the result of me running the code in the last epoch:
```
Epoch 120/120
10000/10000 [==============================] - 9s 875us/step - loss: 0.2077 - acc: 0.9318 - val_loss: 2.7188 - val_acc: 0.5400
```
